integration-webhook/src/main/java/org/symphonyoss/integration/webhook/parser/metadata/MetadataParser.java

```diff
@@ -244,14 +258,64 @@ private void processMetadataObjects(EntityObject root, JsonNode node, List<Metad
   }
 
   /**
+   * Process objects that have N items (a list).
+   * @param node Input JSON containing values to be processed.
+   * @param object Metadata object used to match template with JSON values.
+   * @param entity Where values are created during the process.
+   * @return A list of entity-objects containing the JSON values.
+   */
+  private List processListMetadataObjects(JsonNode node, MetadataObject object,
+      EntityObject entity) {
+    List itemsList = new ArrayList();
+
+    if (node.path(object.getId()) instanceof ArrayNode) {
```

Comment:
I believe you don't need to perform this check and also cast the node object to ArrayNode. You could execute the for-each loop directly.